msal/managed_identity.py

```diff
@@ -563,7 +598,12 @@ def _obtain_token_on_service_fabric(
     logger.debug("Obtaining token via managed identity on Azure Service Fabric")
     resp = http_client.get(
         endpoint,
-        params={"api-version": "2019-07-01-preview", "resource": resource},
+        params={k: v for k, v in {
+            "api-version": "2019-07-01-preview",
```

Comment:
Can you define the API value as a constant?

Reply:
This code base defines constant typically only when it's being used multiple times. If a value is used only once, we tend to use the literal value on the spot. This way, the implementation has better locality; the new reader/maintainer does not need to jump through multiple files (or rely on intellisense in their IDE) to find the real value.

It is a matter of style preference. We can talk more offline.

Reply:
I personally think it would be better to define a constant for something like an API version, but you are the implementer in Python so it's up to you. I'll resolve this.